package.json

```diff
@@ -95,15 +95,14 @@
     "fuse.js": "3.4.4",
     "globs-to-files": "1.0.0",
     "google-closure-compiler": "20190215.0.2",
-    "gulp": "3.9.1",
+    "gulp": "4.0.1",
     "gulp-ava": "1.0.0",
     "gulp-eslint": "5.0.0",
     "gulp-eslint-if-fixed": "1.0.0",
     "gulp-file": "0.4.0",
     "gulp-git": "2.9.0",
-    "gulp-help": "1.6.1",
+    "gulp-help": "chmontgomery/gulp-help#gulp4",
```

Comment:
Will renovate be able to update this dependency correctly?

Reply:
Good question. The next update for this package should be a normal `npm` semver release. See https://github.com/chmontgomery/gulp-help/issues/31